news/2022-10-04-changes-to-osu-mania-sr-and-pp.md

```diff
@@ -0,0 +1,62 @@
+## osu!mania changes
+
+### Long note difficulty changes
+
+A [new change](https://github.com/ppy/osu/pull/17913) in long note calculation fixes issues with fast releases giving disproportionately more star rating than they should.
+
+Before this change, all overlapping long notes would receive a fixed bonus. After this change, this bonus now scales proportionally with how long ago the last long note release was.
+
+This change also fixes an oversight with long notes in the rightmost column of a map taking precedence in overlap calculation.
+
+An [interactive demonstration](https://www.desmos.com/calculator/7eboanfzne) by [**Evening**](https://osu.ppy.sh/users/2193881) helps visualize how this change affects the long note bonus if you're interested.
+
+### Chord difficulty changes
+
+An [issue has been fixed](https://github.com/ppy/osu/pull/19120) where star rating would change depending on the order of notes in the beatmap file.
+
+This change makes sure that the difficulty of the entire chord is taken into account, rather than only the last note to be processed by the difficulty calculator.
+
+This primarily affects maps with heavy usage of chords (especially in higher key modes), for example:
+
+- [Cardboard Box - The Limit Does Not Exist [[7K] Infinity]](https://osu.ppy.sh/beatmapsets/379758#mania/831204): 7.45\* --> 7.71\*
+- [Manabu Namiki - Tenshi (True Final Boss) [[4K] MIPHATHER]](https://osu.ppy.sh/beatmapsets/1503789#mania/3080978): 5.36\* --> 5.44\*
+
+### Using accuracy instead of score for performance points
+
+Overall score has historically been used as the measure of skill a player demonstrates on a given map. With the new direction [osu!(lazer)](https://github.com/ppy/osu) is taking, it no longer makes sense to base performance entirely on this metric.
+
+With a [new change](https://github.com/ppy/osu/pull/18749), awarded pp will now be based on the accuracy the player gets on the map using this formula:
+
+put accuracy formula image here
+
+At 80% accuracy and below, the player will receive no pp for their score. This scales linearly to 100% accuracy where the player receives the maximum possible pp for their play.
+
+Compared to the old system, this buffs very high-accuracy scores while nerfing lower-accuracy ones. However, some scores that previously received no pp will find that they now do reward some pp.
+
+### Other changes
+
+As a consequence of accuracy now determining pp, this has led to some other changes:
+
+- **No Fail now rewards 0.75x of No Mod pp.** Previously, due to the multiplier for this mod limiting score to below 500,000, it was impossible to gain any pp with it. This is no longer the case.
+- **"Accuracy pp" has been removed.** Scores above 960,000 previously awarded a bonus to overall pp gained in a bonus termed as "accuracy pp". This has been removed but accounted for in the new accuracy pp formula described above.
+- **Star rating to pp curve formula has been simplified.** These changes are only minor.
```

Comment:
is this curve formula thing important to mention at all? will the end user see any actual change from this? best to provide an example if they would like the other changes

Reply:
The change is minimal in terms of effect (users will not notice this), but I still worry that a change like this should be documented in a somewhat accesible place. I don't think we have any place to achieve this rather than the newspost

Would a wording in the lines of 

> **Star rating to pp curve formula has been simplified.** This change doesn't have noticeable effects in the pp rewarded to plays, and is only intended to make the calculations more efficient.

work better?

Reply:
That being said, I think that if we do include it, we should describe *how* it was changed (with the formulas themselves) rather than just mentioning the change and not going further

Reply:
I think removing it is best. It's not intended to have any drastic effect on pp calculation and seems to be causing more confusion than its worth. Explaining the difference between 2 very similar formulas won't help much either.

Reply:
i'm pretty sure this point is covered with the graph kominaru posted. the segmented part has been simplified to a linear formula, which i believe is what this is referring to

Reply:
> i'm pretty sure this point is covered with the graph kominaru posted. the segmented part has been simplified to a linear formula, which i believe is what this is referring to

It's not the same formula: the graph in the post above covers the `acc -> % of maximum pp awarded` curve. This point here corresponds to the `sr -> maximum pp awarded` curve instad